client/CVideoHandler.h

```diff
@@ -36,6 +36,7 @@ class IMainVideoPlayer : public IVideoPlayer
 	{
 		return false;
 	}
+	virtual std::pair<std::unique_ptr<ui8 []>, si64> getAudio(const VideoPath & videoToOpen) { return std::pair<std::unique_ptr<ui8 []>, si64>(std::make_pair(std::make_unique<ui8[]>(0), 0)); };

```

Comment:
```suggestion
	virtual std::pair<std::unique_ptr<ui8 []>, si64> getAudio(const VideoPath & videoToOpen) { return std::make_pair(std::make_unique<ui8[]>(nullptr), 0); };
```

Reply:
Is not possible to use `nullptr` here. Accepts only `0`. :(

Reply:
I guess this might work
```
std::make_pair(nullptr, 0);
```